content/broker/broker-transition/getting-started/_index.nb.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: Komme i gang med Formidling Overgangsløsning
+LinkTitle: Komme i gang
+Descriptions: Hvordan komme i gang med å bruke Formidling overgangsløsningen fra Altinn 2 til Altinn 3
+Tags: [Løsning, formidling, overgang, guide]
+TOC: False
+Weight: 1
+---
+
+## Tjenesteeiere
+For å bruke Formidling overgangsløsningen i Altinn 2 for å opprette, laste opp og hente fil metadata, må en tjenesteeier fullføre følgende trinn.
+1. Ha en eksisterende Altinn 2 Formidlingstjeneste.
+2. Ha eller lag en tilsvarende Altinn 3 Formidlings-ressurs.
+Se hvordan du kommer i gang med Altinn 3 Formidling [her](../../getting-started/).
+<!-- TODO: Oppsett av rettigheter for Altinn 3-ressursen-->
+<!-- TODO: Bør vi ha lenker her til de forskjellige tjenestene som brukes av tjenesteeiere for å lage de forskjellige tjenestene?-->
+1. Be om et overgangsoppsett fra Altinn 2 tjenesten til Altinn 3 ressursen. Bestem dato for når dette skal gå i live. (For å minimere risiko for foreldreløse data i Altinn 2 lagring)
```

Comment:
Is this the start of a new list, or is it a continuation of the previous? Currently this is formatted as a single list, but with a numbering 1,2, 1,2,3.
Either change numbering og add some text og header between.